tickets/src/test/java/no/unit/nva/publication/ticket/create/CreateTicketHandlerTest.java

```diff
@@ -680,6 +681,7 @@ private InputStream createHttpTicketCreationRequest(TicketDto ticketDto,
         throws JsonProcessingException {
         return new HandlerRequestBuilder<TicketDto>(JsonUtils.dtoObjectMapper)
                    .withBody(ticketDto)
+                   .withAuthorizerClaim(PERSON_AFFILIATION_CLAIM, userCredentials.getUsername())
```

Comment:
Is it possible to add method withPersonAffilition() ?

Reply:
Not possible.